domains/snnttechv.is-local.org.json

```diff
@@ -0,0 +1,13 @@
+{
+    "domain": "snnttechv.is-local.org",
+    "subdomain": "*snnttechv",
+
+    "owner": {
+        "email": "nyi4xb1@gmail.com"
+    },
+
+    "record": {},   
```

Comment:
Records are required, otherwise your subdomain will not get created.

Reply:
how to find record? Sir

Reply:
You would need to figure that part out, in the readme of this repo you can find an example file for this, and it will show you all the different records you can add.